build/__phpVersions.sh

```diff
@@ -7,7 +7,7 @@ COMPOSER_SETUP_SHA384='906a84df04cea2aa72f40b5f787e49f22d4c2f19492ac310e8cba5b96
 COMPOSER1_9_VERSION='1.9.3'
 COMPOSER1_10_VERSION='1.10.19'
 COMPOSER2_0_VERSION='2.0.8'
-COMPOSER2_2_VERSION='2.2.9'
+COMPOSER2_2_VERSION='2.3.4'
```

Comment:
Can we have COMPOSER2_3_VERSION='2.3.4' and keep the 2.2 one

Reply:
Updated. Thank you